tests/test_micropip.py

```diff
@@ -541,15 +543,14 @@ def test_install_mixed_case2(selenium_standalone_micropip, jinja2):
     )
 
 
-@pytest.mark.xfail(reason="test fails in some environments")
 @pytest.mark.asyncio
 async def test_install_keep_going(mock_fetch: mock_fetch_cls) -> None:
     dummy = "dummy"
     dep1 = "dep1"
     dep2 = "dep2"
     mock_fetch.add_pkg_version(dummy, requirements=[dep1, dep2])
-    mock_fetch.add_pkg_version(dep1, platform="native")
-    mock_fetch.add_pkg_version(dep2, platform="native")
+    mock_fetch.add_pkg_version(dep1, platform="windows")
+    mock_fetch.add_pkg_version(dep2, platform="windows")
```

Comment:
Why windows though? Is the assumption that we won't run this on windows?

Reply:
> Is the assumption that we won't run this on windows?

Right, but probably I should just use a invalid platform string.